src/components/GlobalFilter/GlobalFilter.js

```diff
@@ -22,46 +36,60 @@ const GlobalFilter = ({
     }
 
     return (
-        <div className="global-filter-container">
-            <span> Busca detalhada </span>
-            <input
-                type="search"
-                placeholder="Insira o nome da área"
-                ref={node => {input = node;}}
-                onKeyUp={() => {
-                    handleKeyUpSearch(input.value)
-                }}
-            />
-            <span> Tipo de seleção </span>
+        <div className="global-filter">
             <form>
-                <div className="selectionTypeContainer">
-                    <label htmlFor="selectionType1" className="asdasd">
-                        <input
-                            name="selectionType"
-                            type="radio"
-                            id="selectionType1"
-                            value="borda"
-                            defaultChecked="checked"
-                            onClick={(e) => handleTypeChange(e)}/>
-                        Demarcada
-                    </label>
-                    <label htmlFor="selectionType2" className="asdasd">
+                <fieldset className="global-filter-form">
+                    <p className="global-filter-form--title">Busca detalhada</p>
+                    <fieldset className="global-filter-form--inputsearch">
                         <input
-                            name="selectionType"
-                            type="radio"
-                            id="selectionType2"
-                            value="opaco"
-                            onClick={(e) => handleTypeChange(e)}/>
-                        Isolada
-                    </label>
-                </div>
-                <label> Opacidade da seleção
-                    <input className="opacitySelection" type="range" min="0" max="10" defaultValue="5" onChange={(e) => handleOpacityChange(e)}></input>
-                </label>
+                            id="searchField"
+                            type="search"
+                            placeholder="Insira o nome da área"
+                            ref={node => {input = node;}}
+                            onKeyUp={() => {
+                                handleKeyUpSearch(input.value)
+                            }}
+                        />
+
+                        <label htmlFor="searchField"><i className="fa fa-search"></i></label>
+                    </fieldset>
+                    <fieldset className="global-filter-form--selectiontype">
+                        <label>Tipo de seleção:</label>
+                        <label htmlFor="selectionType1" className="input-checkopacity">
+                            <input
+                                name="selectionType"
+                                type="radio"
+                                id="selectionType1"
+                                value="borda"
```

Comment:
I think this `value` attribute was forgotten, no? Don't look like it makes a difference.

Reply:
Yes, it's right, I'll remove it.